CordovaDemo/platforms/ios/www/plugins/pspdfkit-cordova-ios/PSPDFKitPlugin/pspdfkit.js

```diff
@@ -209,6 +209,11 @@ var PSPDFKitPlugin = new function() {
         importXFDF: ['xfdfPath', 'callback'],
         exportXFDF: ['xfdfPath', 'callback'],
     });
+	
+    // Document Processing
+    addMethods({
+        processAnnotations: ['annotationChange', 'processedDocumentPath', 'callback', 'annotationType'],
```

Comment:
Is there any special reason the callback is not at the end?
I feel like that would be more expected, instead of having the annotation type the last parameter.

Reply:
If we don't pass it, the call back is not called. We do the same for the `options` parameter in `present`: https://github.com/PSPDFKit/Cordova-iOS/blob/master/PSPDFKitPlugin/pspdfkit.js#L103

Reply:
Ah, I see. If we already do it for `present` then it's fine.
But we should revert my suggestion from https://github.com/PSPDFKit/Cordova-iOS/pull/80#discussion_r299477251 in that case.

Reply:
Yeah. You're right! I just reverted it.